src/components/graph-vis/filters.js

```diff
@@ -34,7 +35,7 @@ class Filters extends React.Component {
         label="Elements"
         value={store.filterElements.map((value) => ({ label: value, value }))}
         isMulti
-        options={store.knownElements.map((value) => ({ label: value, value }))}
+        options={store.knownElements.slice().sort(sortStringsLength).map((value) => ({ label: value, value }))}
```

Comment:
please sort it already in the store when computing the `knownElements`

Reply:
Done.